runtime/shibuya/src/lib.rs

```diff
@@ -1380,7 +1380,7 @@ pub type Executive = frame_executive::Executive<
 /// All migrations that will run on the next runtime upgrade.
 ///
 /// Once done, migrations should be removed from the tuple.
-pub type Migrations = ();
+pub type Migrations = (pallet_dapp_staking_v3::migrations::DappStakingV3TierRewardAsTree<Runtime>,);
```

Comment:
might need to rebase this PR or the price provider PR (depending on which merge first)

Reply:
Most definitely 🙂